packages/ra-ui-materialui/src/form/Toolbar.tsx

```diff
@@ -65,19 +66,20 @@ export const Toolbar = <
         children,
         className,
         resource,
-        saving,
-        submitOnEnter = true,
-        mutationMode,
+        saving: saveingProp,
+        submitOnEnter: submitOnEnterProp,
+        mutationMode: mutationModeProp,
         ...rest
     } = props;
     const record = useRecordContext(props);
+    const saveContext = useSaveContext();
```

Comment:
Don't forget to pass the props as they may override the context

Reply:
No, `useSaveContext` doesn`t have this feature like `useRecordContext` have. Should I add it?

Reply:
Then the Toolbar doesn't need to accept the `saving` prop